### ISSUE TITLE
Fermenter HW settings don't match fermentation display

### ISSUE THREAD

**Issue by helibrewer** [Manuel83/craftbeerpi3#28]
Fermentation Tab shows:
Sensor1: S1(temp probe1)
Sensor2: S2(temp probe 2)

Fermenter Hardware shows:
Logic: PIDFermentationChamber
Actor: blank
Cooler: S1 (SSR)
Sensor1: S1
Sensor2: S1
Sensor3: S1

So after creating the Fermentation setup, the Fermenter Hardware in "System->Hardware Settings" does not show the proper configuration. The Fermentation tab DOES reflect the correct setup.

**Comment by mrillies** [comment]
Yep looks like the its in the JavaScript, it just fetches the name of sensor 1 and renders it 3 times.

I'm sure this will be looked at after the 3.1 release
